batch/batch/worker/worker.py

```diff
@@ -1019,6 +1082,7 @@ async def container_config(self):
                 },
             },
             'linux': {
+                'rootfsPropagation': 'slave',
```

Comment:
Why is this not `rslave`?

Reply:
While it probably wouldn't hurt anything, we don't need `rslave` so I went with the more restrictive option. `slave` says that any mount points beneath the root mount will be propagated into the container, but only one level of mounts deep. `rslave` means this is applied recursively to mounts underneath mounts, e.g. if we were to bind mount an additional mount inside the directory tree of one of our fuse mounts, but we don't do this.